components/suggest/src/store.rs

```diff
@@ -166,6 +206,29 @@ where
     fn ingest(&self, constraints: SuggestIngestionConstraints) -> Result<()> {
         let writer = &self.dbs()?.writer;
 
+        if let Some(unknown_records) =
+            writer.read(|dao| dao.get_meta::<UnparsableRecords>(UNKNOWN_RECORDS_META_KEY))?
+        {
+            let all_unparsable_ids = unknown_records
+                .0
+                .iter()
+                .filter(|(_, unparsable_record)| unparsable_record.schema_version < VERSION)
+                .map(|(record_id, _)| record_id)
+                .collect::<Vec<_>>();
+            for unparsable_ids in all_unparsable_ids.chunks(150) {
```

Comment:
suggestion: Let's put this `150` into a constant, please (and give it a docstring), so that our future selves aren't left wondering what it means. We could call it something like `UNPARSABLE_IDS_PER_REQUEST`—how does that sound to you?